CHANGELOG.md

```diff
@@ -1,5 +1,337 @@
 # Changelog
 
+## 🔎 1.1.0
+
+Wrangler 1.1.0 includes a number of improvements to documentation and project stability, including:
+
+- ### Security
+
+  - **Change global config perm - [xtuc], [pull/286]**
+
+  This PR improves the security of Wrangler's global config file, by only allowing the user using Wrangler to view the file. **For existing Wrangler users, please run `wrangler config` again on your machine!** This will fix the permissions of your global config file to be scoped to your user account.
```

Comment:
"This PR improves the security of Wrangler's global config file, by only allowing the user using Wrangler to view the file." -> "This PR improves the security of Wrangler's global config file, by restricting access to Wrangler's global user configuration file (`~/.wrangler/config/default.toml`) to the user who ran the `wrangler config` command."